phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -1058,6 +1054,10 @@ public function format_methodparam($open, $name, $attrs) {
                 }
                 return $content;
         }
+        if ($this->params["opt"] && !$this->params["init"]) {
+            return " = ?</span>";
+        }
```

Comment:
@cmb69 Is it possible to change `return " = ?</span>"` to `return "<span class="initializer"> = ?</span></span>"`?

Reply:
Ah, good point. Has `.initializer` already style rules?

Reply:
Other default values are surrounded with it. e.g.
```
<span class="methodname"><strong>apcu_dec</strong></span>
    ( <span class="methodparam"><span class="type">string</span> <code class="parameter">$key</code></span>
   , <span class="methodparam"><span class="type">int</span> <code class="parameter">$step</code><span class="initializer"> = 1</span></span>
   , <span class="methodparam"><span class="type">bool</span> <code class="parameter reference">&$success</code> = ?</span>
   , <span class="methodparam"><span class="type">int</span> <code class="parameter">$ttl</code><span class="initializer"> = 0</span></span>
   ) : <span class="type"><span class="type">int</span>|<span class="type"><span class="type false">false</span></span></span></div>
```
https://www.php.net/manual/en/function.apcu-dec.php

Reply:
Ah, right, of course! Thanks for pointing this out. Done with c1dbe7b009e9399fa1024df963266ee52d365d62.

Reply:
No problem :) Thanks a lot for your quick fix!